automation/features/pageobjects/page.js

```diff
@@ -50,12 +50,51 @@ class Page {
 
   goTo(path) {
     browser.url(path);
+    browser.waitUntil(
+      function () {
+        const state = browser.execute(function () {
+          return document.readyState;
+        });
+        return state === "complete";
+      },
+      {
+        timeout: 10000,
+        timeoutMsg: "Page did not fully load in 10 seconds",
+      }
+    );
```

Comment:
question (blocking): why is this necessary?

Reply:
helps to alleviate some flakiness if the page is not ready and the tests aim to click on items not yet loaded